### PR TITLE
Feature/items

### PR DESCRIPTION
**Pull request by taka66**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -81,6 +81,8 @@
     "koiki-ui": "0.0.46",
     "less": "^2.5.3",
     "less-loader": "^2.2.1",
+    "node-sass": "^4.5.0",
```
Comment (sideroad) [inline package.json:84]: これいるんだっけ？

File: src/containers/TimelinePage.jsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+                  <title>waterMark</title>
+                  <circle cx="81.27" cy="51.72" r="47.37" />
+                  <path
+                    d="M9.44,155H151.5c-1.29-23.18-9.23-46.35-35-60.94-24.89,
+                    17.17-48.5,16.31-71.25,0C21.67,105.21,11.8,131,9.44,155Z"
+                  />
+                </svg>
+              </td>
+              <td>
+                <p className="item-name">Nanbun Chin</p>
```
Comment (sideroad) [inline src/containers/TimelinePage.jsx:84]: ロシア人として　ナブチェンコフ　を追加していただければと思います。

File: src/containers/TimelinePage.jsx [hunk 3]
```diff
@@ -11,10 +11,10 @@ const TimeLinePage = props => console.log(props) ||
 <div>
   <div>
     <div id="wrapper" className="wrapper">
-      <div className="logo">
+      <div className={styles.logo}>
         <img
           src="images/logo.png"
```
Comment (sideroad) [inline src/containers/TimelinePage.jsx]: this is better to be bundle

File: package.json (hunk 1 above)
Comment (taka66) [reply package.json:84]: sass loaderの野郎が欲しがってるみたいです、、
外したらこんな感じです

npm WARN sass-loader@4.1.1 requires a peer of node-sass@^3.4.2 || ^4.0.0 but none was installed.
[0] [webpack-isomorphic-tools/plugin] [error] Module "./~/css-loader?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]!./~/autoprefixer-loader?browsers=last 2 version!./~/sass-loader?outputStyle=expanded&sourceMap!./~/flag-icon-css/sass/_flag-icon.scss" has no source. Maybe Webpack compilation of this module failed. Skipping this asset.
[0] [webpack-isomorphic-tools/plugin] [error] Module "./~/css-loader?modules&importLoaders=2&sourceMap&localIdentName=[local]___[hash:base64:5]!./~/autoprefixer-loader?browsers=last 2 version!./~/sass-loader?outputStyle=expanded&sourceMap!./~/react-flag-icon-css/lib/sass/main.scss" has no source. Maybe Webpack compilation of this module failed. Skipping this asset.

**Review by sideroad — APPROVED** [review]

**Pull request merged**